elastic4s-core/src/main/scala/com/sksamuel/elastic4s/mappings/attributes.scala

```diff
@@ -508,6 +508,20 @@ object attributes {
     }
   }
 
+  trait AttributeTreeLevels extends Attribute { self: TypedFieldDefinition =>
+
+    private[this] var _treeLevels: Option[Integer] = None
+
+    def treeLevels(treeLevels: Integer): this.type = {
+      _treeLevels = Some(treeLevels)
```

Comment:
Since this is a `java.lang.Integer` it is possible to pass `null` so I think it is better to use `Option(treeLevels)` instead.